bot/exts/pride/pride_leader.py

```diff
@@ -0,0 +1,117 @@
+import json
+import logging
+import random
+from pathlib import Path
+from typing import Optional
+
+import discord
+from discord.ext import commands
+from fuzzywuzzy import fuzz
+
+from bot import constants
+
+log = logging.getLogger(__name__)
+
+PRIDE_LEADERS_RESOURCE = Path("bot/resources/pride/prideleader.json")
+MINIMUM_FUZZ_RATIO = 40
+
+
+class PrideLeader(commands.Cog):
+    """Gives information about Pride Leaders."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+        with PRIDE_LEADERS_RESOURCE.open(encoding="utf8") as data:
+            self.pride = json.load(data)
+
+    def name_verifier(self, leader_name: str) -> Optional[str]:
```

Comment:
This entire function could be replaced with just a `dict.get` inside the command itself.